packages/services/f-analytics/src/services/analytics.mapper.js

```diff
@@ -159,20 +159,21 @@ export const mapUserData = ({ userData, authToken, req } = {}) => {
  * @param {object} pageData - A reference to the current PageData instance
  * @param {string} featureName - The name of the feature
  * @param {string} pageName - The name of the page
- * @param {string} conversationId - The current conversation Id
  * @param {string} requestId - The current request Id
  * @param {number} httpStatusCode - The httpStatusCode (only supplied when 200 needs to be overriden)
+ * @param {object} req - The `request` context
  */
 export const mapPageData = ({
     pageData,
     featureName,
     pageName,
-    conversationId,
     requestId,
-    httpStatusCode
+    httpStatusCode,
+    req
 } = {}) => {
     pageData.group = featureName;
     if (pageName) pageData.name = pageName;
+    const conversationId = getCookie('x-je-conversation', req);
     if (conversationId) pageData.conversationId = conversationId;
     if (requestId) pageData.requestId = requestId;
     if (httpStatusCode) pageData.httpStatusCode = httpStatusCode;
```

Comment:
I know it's not from this PR, but small nit on line 180, `diplay`

Reply:
Also not a massive fan of this method mutating the input object rather than returning a new object, like
```javascript
const conversationId = getCookie('x-je-conversation', req);

return {
    ...pageData,
    group: featureName,
    conversationId: conversationId || pageData.conversationId,
    ... etc, etc
}
```